layouts/reveal/baseof.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="{{ .Site.Language.Lang }}">
+  <head>
+    {{ partial "head.html" . }}
+    {{ partial "revealjs.html" . }}
+  </head>
+  <body>
+    <div class="reveal">
+      <div class="slides">
+        <section data-markdown>
+          <textarea data-markdown>
+            {{ .RawContent }}
+          </textarea>
+        </section>
+      </div>
+      <div class='footer'>
+        <span id="footer-title">{{ .Title }}</span> |
+        Use left and right arrows to navigate. |
+        Press "?" for help. |
+        <a href="/onboarding">Return to onboarding index.</a>
```

Comment:
```suggestion
        <a href="/docs/onboarding">Return to onboarding index.</a>
```
This should point it towards its correct location